tests/integration/targets/gitlab_project_members/tasks/main.yml

```diff
@@ -10,16 +10,19 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 - name: Install required library
-  pip:
+  ansible.builtin.pip:
     name: python-gitlab
     state: present
+    extra_args: >-
+      --break-system-packages
```

Comment:
Why are you adding this?

And it would be leaner as:
```yaml
    extra_args: --break-system-packages
```

Reply:
That's true if you run pip manually, but these are integration tests, not random playbooks/roles. The tests that install requirements are supposed to be run in special environments (`--docker`, `--remote`, or `--venv` parameters to `ansible-test integration`). If you check all `pip` tasks in the community.general integration tests (there are over 80 of them), you will see that none use `--break-system-packages`.